File: C/LadosTriangulo.c
#include <stdio.h>

int main ()
{
    int x,y,z;
    scanf("%d",&x);
    scanf("%d",&y);
    scanf("%d",&z);
    if ( x + y < z || x + z < y || z + y < x )
		{
			printf("NAO FORMA UM TRIANGULO\n");
		}
		else
		{
			if ( x == y && y == z)
			{
				printf("EQUILATERO\n");
			}
			else if ( x != y && y == z || y != y && y == x)
			{
				printf("ISOSCELES\n");
			}
			else
			{
				printf("ESCALENO\n");
			}
		}
    return 0;
}

File: Java/LadosTriangulo.java
import java.util.Scanner;

public class Main 
{
    public static void main ( String [] args )
	{
		Scanner in = new Scanner ( System.in );
		int x,y,z;
		x = in.nextInt();
		y = in.nextInt();
		z = in.nextInt();
		
		if ( x + y < z || x + z < y || z + y < x )
		{
			System.out.println("NAO FORMA UM TRIANGULO");
		}
		else
		{
			if ( x == y && y == z)
			{
				System.out.println("EQUILATERO");
			}
			else if ( x != y && y == z || y != y && y == x)
			{
				System.out.println("ISOSCELES");
			}
			else
			{
				System.out.println("ESCALENO");
			}
		}
		
		in.close();
	}
}
